### PR TITLE
Set up the dependency

### PR DESCRIPTION
**Pull request by chenmoneygithub**
Add some initial dependency of the KerasNLP repo by configuring setup.py.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+# limitations under the License.
+
+"""Setup script."""
+
+from setuptools import find_packages
+from setuptools import setup
+
+setup(
+    name="keras-nlp",
+    description="High-level NLP libraries based on Keras",
```
Comment (mattdangerw) [inline setup.py:22]: Let's sync up this description with keras-cv?
https://github.com/keras-team/keras-cv/blob/master/setup.py

File: setup.py [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+        "Programming Language :: Python :: 3.7",
+        "Operating System :: Unix",
+        "Operating System :: Microsoft :: Windows",
+        "Operating System :: MacOS",
+        "Intended Audience :: Science/Research",
+        "Topic :: Scientific/Engineering",
+        "Topic :: Software Development",
+    ],
+    packages=find_packages(exclude=("tests",)),
+)
```
Comment (mattdangerw) [inline setup.py:43]: nit: newline at end of file

**Review by mattdangerw — APPROVED** [review]
LGTM! Two comments

File: setup.py (hunk 1 above)
Comment (chenmoneygithub) [reply setup.py]: yes! done

File: setup.py (hunk 2 above)
Comment (chenmoneygithub) [reply setup.py]: Yes, done!

**Pull request merged**